libs/portal/features/profile/ui/header.stories.tsx

```
@@ -0,0 +1,17 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { createRemixStub } from '@remix-run/testing';
+import { Header } from './header';
+
+const meta: Meta<typeof Questions> = {
+  title: '🌀 Portal/Profile/Header',
+  component: Questions,
+  tags: ['autodocs'],
+  parameters: {
+    layout: 'centered',
+  },
+};
```

ESLint

[error] Extra semicolon.

(style/semi)